packages/core/src/css/cssConfig.ts

```diff
@@ -0,0 +1,145 @@
+import { createRequire } from 'node:module';
+import path from 'node:path';
+import type { RsbuildConfig, RsbuildPlugin } from '@rsbuild/core';
+import { CSS_EXTENSIONS_PATTERN } from '../constant';
+import { RemoveCssExtractAssetPlugin } from './RemoveCssExtractAssetPlugin';
+
+const require = createRequire(import.meta.url);
```

Comment:
now `require.resolve` is supported to preserve

Reply:
Keep using ESM style require would be better if possible. Use ESM style for all code.

Reply:
> Keep using ESM style require would be better if possible. Use ESM style for all code.

agree but one more thing, if we want to provide the cjs, how would`createRequire` be transformed?

Reply:
done by no change